tests/test_rat_board.py

```diff
@@ -262,3 +262,29 @@ def test_next_free_mixed_board(self, keys: list, expected: int, rat_board_fx: Ra
 
         # and verify its functionality
         assert expected == nextFree
+
+    @pytest.mark.parametrize("test_input", [
+                            "UNIT_TEST",
+                            "@12345678-9876-53d1-ea5e-0000deadbeef",
+                            "42"])
+    def test_search_valid(self, test_input, rat_board_fx: RatBoard, rescue_plain_fx: Rescue):
+        test_rescue = rescue_plain_fx
+        test_rescue.uuid = UUID('12345678-9876-53d1-ea5e-0000deadbeef')
+        rat_board_fx.append(test_rescue)
+        assert rat_board_fx.search(test_input) == test_rescue
+
+    @pytest.mark.parametrize("test_input", [
+                            "unit_tes",
+                            '@12345678-9876-53d1-ea5e-0000dead',
+                            '@12345678-9876-53d1-ea5e-0000deadbee-',
+                            '@12345678-9876-53d1-ea5e-000deadsheep',
+                            42,
+                            "42.5"])
+    def test_search_garbage(self, test_input, rat_board_fx: RatBoard, rescue_plain_fx: Rescue):
```

Comment:
why are you making a shalow copy of the rescue fixture? it is unnecessary and doesn't really do anything.
(leftover from `deepcopy`?)

Reply:
It gives me an "unresolved reference" error if I don't include it.